setup.py

```diff
@@ -14,9 +14,16 @@
     )
     sys.exit(1)
 
+
+def django_guardian_version():
+    if sys.version_info < (3,5):
+        return 'django-guardian<2'
```

Comment:
We do not support django-guardian<1.4.2, so this needs to be updated
```suggestion
        return 'django-guardian>=1.4.2,<2'
```

Reply:
Good catch.  I was thinking in terms of a new installation, but they could already have Django-Guardian installed with a version < 1.4.2 and my code would have missed it.